requirements.txt

```diff
@@ -5,5 +5,5 @@ gunicorn==19.6.0
 pika==0.10.0
 PyJWT==1.4.2
 requests==2.12.4
-sdx-common==0.7.0
+sdx-common==0.7.2
```

Comment:
Should all these requirements be hashed in line with other repos?

Reply:
Should they be hashed on the master or 2.0?

Reply:
Certainly in the 2.0 version - if this is the old version then it's not so vital as it'll go away anyhow.